src/components/ControlsToggles/__tests__/Selection.test.tsx

```diff
@@ -0,0 +1,461 @@
+        fontFamily: 'Lato-Regular',
+      })
+    );
+
+    expect(getByTestId('Selection.Item.Pressable-item-3').props.style[1]).toEqual(
+      expect.objectContaining({
+        backgroundColor: colors.moon100,
+      })
+    );
+    expect(getByText('Item #3').props.style).toEqual(
+      expect.objectContaining({
+        color: colors.moon200,
+        fontFamily: 'Lato-Regular',
+      })
+    );
+
+    expect(getByTestId('Selection.Item.Pressable-item-4').props.style[1]).toEqual(
+      expect.objectContaining({
+        backgroundColor: colors.space100,
+      })
+    );
+    expect(getByText('Item #4').props.style).toEqual(
+      expect.objectContaining({
+        color: colors.moon900,
+        fontFamily: 'Lato-Regular',
+      })
+    );
+
+    expect(getByTestId('Selection.Item.Pressable-item-5').props.style[1]).toEqual(
+      expect.objectContaining({
+        backgroundColor: colors.space100,
+      })
+    );
+    expect(getByText('Item #5').props.style).toEqual(
+      expect.objectContaining({
+        color: colors.moon900,
+        fontFamily: 'Lato-Regular',
+      })
+    );
+
+    expect(getByTestId('Selection.Item.Pressable-item-6').props.style[1]).toEqual(
+      expect.objectContaining({
+        backgroundColor: colors.space100,
+      })
+    );
+    expect(getByText('Item #6').props.style).toEqual(
+      expect.objectContaining({
+        color: colors.moon900,
+        fontFamily: 'Lato-Regular',
+      })
+    );
+  });
+
+  it('renders correctly when backgroundColor and activeItemColor are not passed', () => {
+    const { getByTestId, getByText } = render(
+      <Selection
+        items={props.items}
+        onChange={props.onChange}
+        textColor={props.textColor}
+        activeTextColor={props.activeTextColor}
+        disabledColor={props.disabledColor}
+      />
+    );
+
+    // Check container styles
+    expect(getByTestId('Selection').props.style).toEqual(
+      expect.objectContaining({
+        width: '100%',
+        flex: 0,
+        padding: 4,
+      })
+    );
+
+    // Check Pressable styles
+    expect(getByTestId('Selection.Item.Pressable-item-1').props.style[0]).toEqual(
+      expect.objectContaining({
+        minWidth: '80%',
+        marginVertical: 10,
+        flexDirection: 'row',
+        alignItems: 'center',
+        justifyContent: 'center',
+        flex: 0,
+      })
+    );
+
+    expect(getByTestId('Selection.Item.Pressable-item-1').props.style[1]).toEqual(
+      expect.objectContaining({
+        backgroundColor: colors.uranus500,
+        paddingHorizontal: 24,
+      })
+    );
+    expect(getByText('Item #1').props.style).toEqual(
+      expect.objectContaining({
+        color: colors.space100,
+        fontFamily: 'Lato-Regular',
+      })
+    );
+  });
+
+  it('renders icon aside when option has it', () => {
+    const { queryByTestId } = render(<Selection {...props} />);
+
+    expect(queryByTestId('Selection.Item.Icon-item-1')).not.toBeNull();
```

Comment:
Poderia testar de forma positiva não?
```suggestion
    const { getByTestId } = render(<Selection {...props} />);

    expect(getByTestId('Selection.Item.Icon-item-1')).toBeTruthy();
```